tests/create_attachment/prereq/main.tf

```diff
@@ -8,11 +8,15 @@ provider aws {
 
 provider aws {
   region  = "us-east-1"
-  profile = "owner"
   alias   = "owner"
+  profile = "owner"
```

Comment:
Please add to the documentation that a user who is looking to test the module will need to have a profile called `owner`

Reply:
That was already in the readme, but I went ahead and made a tiny adjustment to the formatting.